app/models/web_hook_event.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class WebHookEvent < ActiveRecord::Base
+  scope :successful, -> { where("status >= 200 AND status <= 299") }
+  scope :failed, -> { where("status > 299") }
```

Comment:
If failed is anything above 299, then shouldn't successful be anything lets say above 0?

Reply:
I think it depends on whether we would call informational responses (100 - 199) as successful or not.